samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -79,6 +79,18 @@ Recovery mode
 In the provisioned state, you can enter the recovery mode for a limited time.
 This mode allows the Android device to recover a lost provisioning key from the locator tag.
 
+Motion detector mode
+--------------------
+
+In the provisioned state, the accessory can activate the motion detector mode from the DULT specification.
```

Comment:
This sounds a bit strange, "activate the motion detector mode **from the ... spec**."
Should we rephrase?

Reply:
maybe "... mode defined in the DULT specification"?